packages/engine/src/lib/handler/flow-executor.ts

```diff
@@ -77,20 +77,14 @@ export const flowExecutor = {
         break;
       }
 
-      const isNotNested = flowExecutionContext.currentPath.path.length === 0;
-      const sendContinuousProgress =
-        isNotNested &&
-        constants.progressUpdateType === ProgressUpdateType.TEST_FLOW;
-      if (sendContinuousProgress) {
-        progressService
-          .sendUpdate({
-            engineConstants: constants,
-            flowExecutorContext: flowExecutionContext,
-          })
-          .catch((error) => {
-            logger.error('Error sending progress update', error);
-          });
-      }
+      progressService
```

Comment:
I see that we're doing this after "if (flowExecutionContext.verdict !== ExecutionVerdict.RUNNING) {", so we won't get to it if it was failed or paused, no?